src/plugins/blocksmith/components/facebook-post-block-modal/index.jsx

```diff
@@ -58,10 +58,20 @@ export default class FacebookPostBlockModal extends React.Component {
     this.handleChangeAside = this.handleChangeAside.bind(this);
   }
 
+  handleMouseOut = e => {
+    e.stopPropagation();
+  };
+
   componentDidMount() {
     setTimeout((t) => {
       t.inputElement.focus();
     }, 0, this);
+    
+    window.addEventListener('mouseout',(e) => this.handleMouseOut(e), true);
```

Comment:
@digitalartcrew this is still an anonymous function.. do something like 

`window.addEventListener('mouseout', this.handleMouseOut, true);`